### ISSUE TITLE
vbs dependancy has been deleted

### ISSUE THREAD

**Issue by jc21** [variadico/noti#97]
#### Description
`http://github.com/variadico/vbs` not found when installing deps

#### Use case
building your package

#### Steps to reproduce
build your package

#### Expected behavior
successful build

#### Actual behavior
```bash
dep ensure

Solving failure: 
        (1) failed to list versions for https://github.com/variadico/vbs: fatal: could not read Username for 'https://github.com': terminal prompts disabled
: exit status 128
        (2) failed to list versions for ssh://git@github.com/variadico/vbs: Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
: exit status 128
        (3) failed to list versions for git://github.com/variadico/vbs: fatal: remote error: 
  Repository not found.
: exit status 128
        (4) failed to list versions for http://github.com/variadico/vbs: fatal: could not read Username for 'https://github.com': terminal prompts disabled
: exit status 128
```

#### Version
golang 1.12

#### Side note
go module support would be nice

**Comment by variadico** [comment]
Noti uses a vendor directory. Unfortunately, modules still have some bugs with vendoring, which aparently may not be fixed until Go 1.14. :cry: 

In order to build this project with Go 1.12, you need to set an environment variable.

```
export GOFLAGS="-mod=vendor"
make build
```

If this has impacted you, the Go team would like to hear an experience report. Maybe you could comment on one of these tickets or create a new one.

* https://github.com/golang/go/issues/27227
* https://github.com/golang/go/issues/32502

The whole point of vendoring is to avoid situations like these. It sucks that modules have broken this behavior.

By the way, I've moved over the dev branch to modules. You still have the same problem, but if you need to build from source, then you can just do `make build`. https://github.com/variadico/noti/tree/dev

Otherwise, if you just need a binary, then you can grab one from the releases page.
https://github.com/variadico/noti/releases

**Comment by variadico** [comment]
I'm gonna close this for now because there's not much else we can do until 1.14. The code does build when you set the vendor flag and modules is coming in noti 3.2.0.
